Automattic/wp-calypso
# Domain Management i1: Rename and rearrange columns and make layout wider

Closes https://github.com/Automattic/dotcom-forge/issues/3738
Closes https://github.com/Automattic/dotcom-forge/issues/3736
## Proposed Changes

* Rename `register_until` column to `Expires/renews on`
* make the domains layout `fullWidth` 
## Testing Instructions



* browse `domains/manage?flags=domains/management`
* Verify presence of "Expires / renews on" column
* Verify wider layout matches design p9Jlb4-8DU-p2

![Screenshot 2023-09-13 at 5 00 26 PM](https://github.com/Automattic/wp-calypso/assets/47489215/b5bce3d1-5159-44e7-8180-e8f45f127382)



## Pre-merge Checklist
